condor_watch_q.py

```diff
@@ -499,7 +508,7 @@ def strip_empty_columns(rows):
     return headers, rows
 
 
-def row_data_from_job_state(clusters):
+def row_data_from_job_state(clusters, summary):
```

Comment:
I'm not sure I like building up the summary information while processing each individual row inside this function. I think I'd rather we built the summary up inside `table_by()` itself... although I'd prefer if `table_by()` only produced the table itself, not any other lines we add.

For now, I recommend moving the summary creation into `table_by` itself, without passing it down into `row_data_from_job_state()`. We can fix the other problem (`table_by()` doing too much) later.